lib/Screens/Login.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class SignIn extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
I think it should be better to keep everything inside Scaffold, no need to wrap Scaffold in paddings/containers. Just move these widget inside Scaffold